spaces/golff/skin.scss

```
@@ -0,0 +1,9 @@
+.yam {
```

The css selector should similar to the space key.
```suggestion
.golff {
```